app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1112,6 +1101,19 @@ def validate_feature_value(self, value):
                 raise serializers.ValidationError(f"Invalid JSON: {e.msg}")
         return value
 
+    def validate(self, data):
+        if data.get("feature_enabled") and not data.get("feature_value"):
+            raise serializers.ValidationError(
+                {"feature_value": NimbusConstants.ERROR_BRANCH_NO_VALUE}
+            )
+
+        if data.get("feature_value") and not data.get("feature_enabled"):
+            raise serializers.ValidationError(
+                {"feature_enabled": NimbusConstants.ERROR_BRANCH_NO_ENABLED}
+            )
+
+        return data
```

Comment:
making this it's own `validate` method 👏

Reply:
Well not just that but also, it was happening at save time, which it doesn't really need to do, but also now we actually can't do this validation at all at this time since we need to force enabled to True for mobile experiments, and we can either do that from the frontend or from the backend, and I tried both and backend was easier/better, so the frontend thinks feature enabled is false and sends that and a value over and it's the backend that forces enabled to True, so if we try to do this validation at save them it'll always fail for mobile experiments.  But there was no reason to do it at save time so I moved it to review time.